### PR TITLE
Change timestamp for all types

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
main issue : [#48988](https://github.com/SekoiaLab/platform/issues/48988)

### CODE REVIEW

File: Palo Alto Networks/paloalto-ngfw/ingest/parser.yml [hunk 1]
```diff
@@ -184,7 +184,7 @@ pipeline:
     external:
       name: dsv.parse-dsv
       properties:
-        input_field: original.message
+        input_field: "{{ original.message.replace('\\\\\\\"', '\\\"') }}"
```
Comment (squioc) [inline Palo Alto Networks/paloalto-ngfw/ingest/parser.yml:187]: Why did you add this replacement?

Comment (TOUFIKIzakarya) [reply Palo Alto Networks/paloalto-ngfw/ingest/parser.yml:187]: There was a problem in the `globalprotect_csv_2.json` event, The value `Microsoft Windows 10 Enterprise, 64-bit` is divided into two, causing a column misalignment. and this's because of `\\\\\\\"`.

**Review by squioc — APPROVED** [review]
